wdfn-server/waterdata/views.py

```diff
@@ -21,25 +22,79 @@
 
 NWIS = NwisWebServices(app.config['SERVER_SERVICE_ROOT'], app.config['SITE_SERVICE_CATALOG_ROOT'])
 
-
 @app.route('/')
 def home():
     """Render the home page."""
     return render_template('index.html', version=__version__)
 
 
-@app.route('/questions-comments/<email>/', methods=["GET", "POST"])
-def questions_comments(email):
+def create_message(target_email, form_data, user_system_data, timestamp):
+    print('timestamp ', timestamp)
+    msg = EmailMessage()
+    msg['Subject'] = 'User Question/Comment for {}'.format(form_data['monitoring-location-url'])
+    msg['From'] = 'WDFN Comments and Questions'
+    msg['Reply-To'] = form_data['user-email-address']
+    msg['To'] = target_email
+
+    message_body = f"""
+            From: {form_data['user-name'] if form_data['user-name'] else 'Name not given'}
+            Subject: {form_data['subject'] if form_data['subject'] else 'No Subject'}
+            Location: {form_data['monitoring-location-url']}
+            Time (UTC): {timestamp}
+            *********** Message ***********
+            {form_data['message']}
+            *********** User Information ***********
+            Email: {form_data['user-email-address']}
+            Phone: {form_data['user-phone-number'] if form_data['user-phone-number'] else 'None given'}
+            Organization or Address: {form_data['user-organization-or-address']
+    if form_data['user-organization-or-address'] else 'None given'}
```

Comment:
This formatting seems funky.

Reply:
True. The IDE did that, and I figured it knew what it was doing. But, now that you point it out, I'm actually surprised if that really did work. Changed it.